app/models/host/discovered.rb

```diff
@@ -9,8 +9,6 @@ class Host::Discovered < ::Host::Base
   belongs_to :hostgroup
   has_one    :discovery_attribute_set, :foreign_key => :host_id, :dependent => :destroy
 
-  validates :mac, :uniqueness => true, :mac_address => true, :presence => true
-  validates :ip, :format => {:with => Net::Validations::IP_REGEXP}, :uniqueness => true
```

Comment:
You can't keep it, there's no ip or mac column in develop anymore, you get errors on validation (and really weird errors at that...)